larq_compute_engine/mlir/python/converter_test.py

```diff
@@ -1,25 +1,26 @@
 import sys
-import pytest
+import unittest
+from unittest import mock
+
 import larq_zoo as lqz
-from tensorflow.python.eager import context
 
-from larq_compute_engine.mlir.python.converter import convert_keras_model
+sys.modules["larq_compute_engine.mlir._graphdef_tfl_flatbuffer"] = mock.MagicMock()
 
+from larq_compute_engine.mlir.python.converter import convert_keras_model
+from larq_compute_engine.mlir._graphdef_tfl_flatbuffer import (
+    convert_graphdef_to_tflite_flatbuffer as mocked_converter,
+)
 
-@pytest.fixture
-def eager_mode():
```

Comment:
How come you don't need eager mode anymore?

Reply:
Good catch, added in afb97556640c0ea111b494c6c312acb68ecedeba